app/assets/stylesheets/layout.scss

```diff
@@ -48,4 +48,41 @@
 
 .text-right {
   text-align: right
+}
+
+a.secondary-button, button.secondary-button {
```

Comment:
This selector could be written as

.secondary-button {
  (styles from line 54-64)
  &:hover,
  &:focus {
    (all other styles)
  }
}

There's no need to prefix with extra specificity with `a` or `button` if they are using the same class. The only reason to include these elemental prefixes would be if other elements are using the class `.secondary-button` that you don't want to apply these styles to.

Reply:
Sure :) will update. I just copied it from the third party developer frontend project